features/api/metrics/index.feature

```diff
@@ -16,37 +16,40 @@ Feature: Product > Metrics index
       | Amatriciana |
     And I go to the metrics and methods page
 
-  Scenario: Default tab is methods
-    Then I should see the following methods:
-      | Margherita  |
-      | Diavola     |
-      | Carbonara   |
-      | Amatriciana |
+  Rule: Tab methods
+    Scenario: Default tab is methods
+      Then I should see the following methods:
+        | Margherita  |
+        | Diavola     |
+        | Carbonara   |
+        | Amatriciana |
+
+    Scenario: New method button
+      Then I should see "Add a method"
+
+    Scenario: Methods and mapping rules
+      Given method "Diavola" is not mapped
+      And method "Carbonara" is mapped
+      When I change to tab "Methods"
+      Then I should be able to add a mapping rule to "Diavola"
+      But I should not be able to add a mapping rule to "Carbonara"
```

Comment:
This is confusing for one who is not looking at the UI. Technically, you _should_ be able to map "Carbonara", even when it's already been mapped. It's only the link that changes from "Add a mapping rule" to a "✓" sign.